.editorconfig

```diff
@@ -1,34 +1,21 @@
 # top-most EditorConfig file
 root = true
 
-# Unix-style newlines with a newline ending every file
+# Unix-style newlines with a newline ending every file, and with same defaults
```

Comment:
Typo: should be "sane".  Unless you mean to say that every file type will have the same defaults, which kind-of goes without saying.

Reply:
Fixed.